test/externalTests/common.sh

```diff
@@ -92,10 +93,11 @@ function force_truffle_version
 
 function truffle_setup
 {
-    local repo="$1"
-    local branch="$2"
+    local soljson="$1"
```

Comment:
Note to self: truffle_setup now accepts three inputs instead of two

Reply:
I'm assuming this won't a big problem because the use of these functions is pretty localized. If I were attempting to use them I'd copy and existing call and modify it anyway.